inst/examples/keras/train.R

```diff
@@ -19,7 +19,7 @@ y_train <- to_categorical(y_train, 10)
 y_test <- to_categorical(y_test, 10)
 
 # set learning_pahse to avoid predictions to require this placeholder
```

Comment:
Let's change this comment to: "required in order to call export_savedmodel on keras models"